lib/ReactViews/Custom/Chart/FeatureInfoPanelChart.jsx

```diff
@@ -54,15 +56,30 @@ class FeatureInfoPanelChart extends React.Component {
   }
 
   render() {
-    if (!ChartableMixin.isMixedInto(this.props.item)) return null;
-    if (!this.chartItem) return null;
+    const catalogItem = this.props.item;
+    const height = this.props.height || this.props.parentHeight;
+    const width = this.props.width || this.props.parentWidth;
+    if (!ChartableMixin.isMixedInto(catalogItem)) {
+      return null;
```

Comment:
should this silently fail or should this throw an error?

Reply:
Good point. I have fixed it now.